src/tests/multihost/alltests/test_misc.py

```diff
@@ -480,3 +479,57 @@ def test_0008_1636002(multihost, backupsssdconf):
         assert "Failed to connect to '/var/lib/sss/db/config.ldb'" \
             not in log_str
         assert "The confdb initialization failed" not in log_str
+
+    @staticmethod
+    @pytest.mark.tier1
+    def test_0009_dbus_method_find_usrby_attr(multihost, backupsssdconf, ldap_posix_usergroup):
+        """
+        :title: D-Bus method to find user by attributes
+        :id: ee3437ff-572e-472e-8f55-0d7d8134266c
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2128840
+        :customerscenario: true
+        :setup:
+          1. In sssd.conf
+             set ldap_user_extra_attrs = sn:sn in domain section of sssd.conf
+             set user_attributes = +sn in [ifp] section
+          2. Create 10 users with 'sn' attribute containing pattern foo
+          3. Create a user who does not have sn=foo pattern in it's sn attribute
+        :steps:
+          1. Restart sssd with clean cache
+          2. Fetch users with attribute 'sn:foo*' with dbus-send command
+          3. Confirm dbus-send command output has all users with foo*
+          4. Confirm dbus-send command output does not contain user
+             who donot have sn=foo* matching pattern
+        :expectedresults:
+          1. SSSD should start successfully
+          2. Command should be completed successfully
+          3. Expected users with sn:foo* are returned
+          4. Expected users without sn:foo* are not returned
+        :teardown:
+          1. Delete users and groups created for test
+          2. Restore sssd.conf
+        """
+        usr = ldap_posix_usergroup
+        client = sssdTools(multihost.client[0])
+        domain_name = client.get_domain_section_name()
+        domain_params = {'ldap_search_base': ds_suffix,
+                         'auth_provider': 'ldap',
+                         'id_provider': 'ldap',
+                         'ldap_uri': f'ldaps://{multihost.master[0].sys_hostname}',
+                         'ldap_tls_cacert': '/etc/openldap/cacerts/cacert.pem',
+                         'use_fully_qualified_names': 'True',
+                         'debug_level': '9',
+                         'ldap_user_extra_attrs': 'sn:sn'}
+        client.sssd_conf(f'domain/{domain_name}', domain_params)
+        client.sssd_conf('ifp', {'user_attributes': '+sn'}, action='add')
+        client.clear_sssd_cache()
+        multihost.client[0].run_command('cp /ets/sssd/sssd.conf /etc/sssd/sssd.conf_ori', raiseonerr=False)
```

Comment:
This does not seem to be useful as it is not restored and there is already backupsssdconf .

Reply:
In addition, the file is being copied **after** being modified, so it is no longer the original file.